package.json

```diff
@@ -100,6 +100,7 @@
     "rollup-plugin-babel": "^4.4.0",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
+    "sass": "^1.77.8",
```

Comment:
would also need to remove node-sass, but didn't need to for the proof of concept